runtime/str.go

```diff
@@ -748,6 +748,29 @@ func strStr(_ *Frame, o *Object) (*Object, *BaseException) {
 	return NewStr(toStrUnsafe(o).Value()).ToObject(), nil
 }
 
+func strSwapCase(f *Frame, args Args, kwargs KWArgs) (*Object, *BaseException) {
+	expectedTypes := []*Type{StrType}
+	if raised := checkMethodArgs(f, "swapcase", args, expectedTypes...); raised != nil {
```

Comment:
You can pass Strtype directly as an argument rather than expanding the expectedTypes slice this way:

```
if raised := checkMethodArgs(f, "swapcase", args, StrType); raised != nil {
```